File: an2_sem4/MPP/Homework/lab3/cs/FestivalSellpoint/FestivalSellpoint.Repo/IBiletRepo.cs
using FestivalSellpoint.Domain;
using System.Collections.Generic;

namespace FestivalSellpoint.Repo
{
    public interface IBiletRepo : IRepo<int, Bilet>
    {
        IEnumerable<Bilet> getBySpectacol(Spectacol spectacol);
    }
}

File: an2_sem4/MPP/Homework/sapt9/java/repo/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/repo/IBiletRepo.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.repo;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Bilet;
import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Spectacol;

public interface IBiletRepo extends IRepo<Integer, Bilet> {
    Iterable<Bilet> getBySpectacol(Spectacol spectacol) throws EntityRepoException;
}
